src/components/ha-labeled-slider.html

```diff
@@ -65,6 +65,10 @@
         type: Number,
         notify: true,
       },
+
+      ignoreBarTouch: {
```

Comment:
`ignoreBarTouch: Boolean`

Reply:
I had that first, but then I just copied the syntax from the other properties to keep it consistent. 
But I guess I could have better fixed them all immediately. :)